pkg/cmd/roachprod/main.go

```diff
@@ -2109,3 +2110,19 @@ The logs will be placed in the directory if specified or in the directory named
 			fetchLogsTimeout)
 	}),
 }
+
+var getLatestPprof = &cobra.Command{
+	Use:   "get-latest-pprof <cluster> [flags]",
+	Short: "downloads the latest pprof file before the provided time.",
```

Comment:
Can we be more precise on what "before" actually means? Is it `<= now` or `< now`?